Dockerfile

```diff
@@ -53,31 +69,21 @@ RUN curl -L -o /usr/local/bin/kops https://github.com/kubernetes/kops/releases/d
 
 ARG ISTIO_VERSION=1.6.2
 RUN curl -L https://istio.io/downloadIstio | ISTIO_VERSION="$ISTIO_VERSION" sh - && \
-    mv /go/istio-$ISTIO_VERSION/bin/istioctl /usr/local/bin/istioctl
+    mv /istio-$ISTIO_VERSION/bin/istioctl /usr/local/bin/istioctl
 
 # Install edgectl
 RUN curl -fL https://metriton.datawire.io/downloads/linux/edgectl -o /usr/local/bin/edgectl && \
     chmod a+x /usr/local/bin/edgectl
 
-# On some images "sh" is aliased to "dash" which does not support "set -o pipefail".
-# We use the "exec" form of RUN to delegate this command to bash instead.
-# This is all because we have a pipe in this command and we wish to fail the build
-# if any command in the pipeline fails.
-ARG POETRY_VERSION=1.0.10
-RUN ["/bin/bash", "-c", "set -o pipefail && curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python"]
-ENV PATH="/root/.poetry/bin:${PATH}"
-COPY poetry.toml /root/.config/pypoetry/config.toml
-
 # Add datawire helm repo
 RUN helm repo add datawire https://www.getambassador.io
 
 WORKDIR /root/aladdin
 
-# Install aladdin python requirements
-COPY pyproject.toml poetry.lock ./
-COPY lib/build-components /root/aladdin/lib/build-components
-RUN poetry install --no-root
-
+COPY --from=code /root/.poetry /root/.poetry
+COPY --from=code /root/.venv /root/.venv
```

Comment:
why are we bothering with `.venv`?

Reply:
it makes docker multistage easier, since we know everything is in that dir